### ISSUE TITLE
Nativescript related code in no-nativescript platforms

### ISSUE THREAD

**Issue by hoisel** [nstudio/xplat#28]
Hi,
Why are *nativescript* packages added to package.json when I create a workspace with web/ionic platforms?

package.json:
```json
    "nativescript-angular": "~6.1.0",
    "nativescript-ngx-fonticon": "^4.2.0",
    "nativescript-theme-core": "^1.0.4"
```
Is this a bug?

**Comment by NathanWalker** [comment]
Definitely a bug, thank you for reporting!

**Comment by NathanWalker** [comment]
Fixed in `6.2.6` 👍
